docs/writing-rules/data-flow/taint-mode.md

```diff
@@ -306,3 +630,104 @@ Semgrep can also treat metavariables with the same name as the _same_ metavariab
 The following example demonstrates the use of source and sink metavariable unification:
 
 <iframe src="https://semgrep.dev/embed/editor?snippet=G652" border="0" frameBorder="0" width="100%" height="432"></iframe>
+
+## Inter-procedural analysis (Pro)
```

Comment:
```suggestion
## Inter-procedural analysis (Pro)
```
Should we call this "cross-function"?

Reply:
I'd prefer we just add a clarification in the text like "also referred as cross-function" or something. I see "cross-function" as a more marketing friendly term targeting those who will actually buy, whereas I see these docs as oriented towards rule writers or those who will actually use the tool. If you google both terms you see that "interprocedural analysis" gives you very relevant results, whereas "cross-function analysis" gives you mostly unrelated results.